src/txmempool.cpp

```diff
@@ -20,9 +20,10 @@
 
 CTxMemPoolEntry::CTxMemPoolEntry(const CTransactionRef& _tx, const CAmount& _nFee,
                                  int64_t _nTime, unsigned int _entryHeight,
-                                 bool _spendsCoinbase, int64_t _sigOpsCost, LockPoints lp):
-    tx(_tx), nFee(_nFee), nTime(_nTime), entryHeight(_entryHeight),
-    spendsCoinbase(_spendsCoinbase), sigOpCost(_sigOpsCost), lockPoints(lp)
+                                 bool _spendsCoinbase, int64_t _sigOpsCost, LockPoints lp): 
+lockPoints(lp), tx(_tx), nFee(_nFee)
```

Comment:
In commit "Re-pack the CTxMempoolEntry to be closer to optimally packed"

Unusual indentation here, maybe reformat.